packages/disploy/src/structs/PartialGuildMember.ts

```diff
@@ -37,13 +44,18 @@ export class PartialGuildMember extends Base {
 	 * Timestamp of when the time out will be removed; until then, they cannot interact with the guild.
 	 */
 	public communicationDisabledUntil!: boolean | null;
-
-	public constructor(app: App, raw: APIGuildMember | APIInteractionDataResolvedGuildMember) {
-		super(app);
+	/**
+	 * The raw data of the member.
+	 */
+	declare public raw: Partial<APIGuildMember | APIInteractionDataResolvedGuildMember>;
+	public constructor(app: App, raw: Partial<APIGuildMember | APIInteractionDataResolvedGuildMember> & { id?: Snowflake }) {
```

Comment:
Maybe don't use a Partial here, since a lot of properties such as `id` are never missing from the API.